friends/src/components/FriendsForm.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const FriendsForm = () => {
+  return(
+    <div>
+      <h2>New Friends, Welcome!</h2>
+      <form>
+        <input type="text" placeholder="enter name" />
```

Comment:
I usually break down my input into multiple lines since you're going to have more than 2 in the end.

Reply:
true. will do today